src/definitions/modules/search.js

```diff
@@ -1538,8 +1538,17 @@
                             html += '<div class="results">';
                             $.each(category.results, function (index, result) {
                                 html += result[fields.url]
-                                    ? '<a class="result" href="' + result[fields.url].replace(/"/g, '') + '">'
-                                    : '<a class="result">';
+                                    ? '<a href="' + result[fields.url].replace(/"/g, '') + '" '
+                                    : html += '<div ';
+
+                                html += result[fields.id] !== undefined
+                                    ? ' id="' + result[fields.id] + '" '
```

Comment:
Needs to be sanitzed (doubple quotes removed
```suggestion
                                    ? ' id="' + result[fields.id].replace(/"/g, '') + '" '
```